<ui version="4.0" >
 <class>Widgets_Graph</class>
 <widget class="QWidget" name="Widgets_Graph" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="FigureCanvas" name="canvas" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="GraphNavigationBar" name="navigationBar" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphNavigationBar</class>
   <extends>QWidget</extends>
   <header>Widgets.h</header>
  </customwidget>
  <customwidget>
   <class>FigureCanvas</class>
   <extends>QWidget</extends>
   <header>Widgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
